source/_posts/vue-security-best-practices.md

```diff
@@ -15,10 +15,10 @@ Hi, #VueFriend! In this post, we cover security best practices around the Vue ec
 
 <!-- toc -->
 
-## "v-if" vs "v-show", don't have the same logic
-Sometimes beginner Vue developers confuse the v-if and v-show directives or use them indifferently. The "v-if" lets you render or not some HTML portion through some logic condition. Otherwise, the v-show always renders the HTML portion through some logic condition too, but it displays or hides the content using the "display: none" CSS property. The HTML portion is always reflected on the HTML template. In summary "v-if" renders or not, and "v-show" always renders, but hides or displays the content.
+## `v-if` vs `v-show`, don't have the same logic
```

Comment:
@cr0wg4n maybe better "`v-if` vs `v-show`,  they don't have the same logic" ? Grammar :)